README.rst

```diff
@@ -166,6 +172,11 @@ Goals for the project
 - We would love to see the Airflow community members create, maintain and share their providers to build an Ecosystem
   of Providers.
 
+Limitations
+-----------
+
+- Async sensors take param ``mode`` but it doesn't have any usage in deferrables.
```

Comment:
did not understand this, can you elaborate please?

Reply:
yes, sure I will add more details to it.

Reply:
i have changed it .